index.html

```diff
@@ -6,11 +6,49 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Kevin Salina Portfolio</title>
-    <link rel="stylesheet" href="/styles/styles.css">
+    <link rel="stylesheet" href="styles/styles.css">
+    <script src="https://kit.fontawesome.com/421a2e0bfb.js" crossorigin="anonymous"></script>
 </head>
 
 <body>
 
+    <header>
+        <div class="header-background-img">
+            <div class="profile-img"></div>
+            <h1 class="header-title">Web Developer / Software Engineer</h1>
+            <button class="btn call-to-action-btn">Contact me</button>
+        </div>
+    </header>
+
+    <section class="container about-me">
+        <p class="about-me-text">
+            Hello there! Ny name is <span>Kevin Salina</span> and I'm a web and software developer. I love problem
+            solving, and bringing people's ideas to life on the web! I am skilled in Javascript, Node, Express, HTML,
+            and CSS. I also love to golf, snowboard, and listen to great alternative rock music.
+        </p>
+    </section>
+
+    <section class="container projects">
+        <h2 class="projects-title">Projects</h2>
+        <p class="projects-text">Loading... please come back soon!</p>
+    </section>
+
+    <section class="container contact-me">
+        <div class="social-icon"><a class="li" href="https://www.linkedin.com/in/kevinsalina25/" target="_blank"
```

Comment:
me being picky, but can possibly format this a little better

Reply:
funny github doesnt show my sample properly, but you can try editing the comment above and see what i mean